packages/paste-core/components/table/src/types.ts

```diff
@@ -62,9 +62,8 @@ export interface TrProps extends React.TableHTMLAttributes<HTMLTableRowElement>
   children: NonNullable<React.ReactNode>;
   element?: BoxProps['element'];
   verticalAlign?: TableVerticalAlignmentOptions;
-  isLastRow?: boolean;
```

Comment:
question: is there a chance it was being used by someone even if it was doing nothing? if so would we want to think twice about removing it to avoid giving someone a new ts error? sidenote: why did we add it in the first place 🤔